lib/ripper-tags/vim_formatter.rb

```diff
@@ -59,6 +59,14 @@ def display_inheritance(tag)
       end
     end
 
+    def display_line_number(tag)
+      if options.line_number && tag[:line]
+        "\tline:%s" % tag[:line]
```

Comment:
Do you mind adding support for both options, since you're already halfway already? Let's support `--excmd=number` as well as `--fields=+n`. Please process the characters within `--fields` with the [same mechanism as `--extra`](https://github.com/tmm1/ripper-tags/blob/a1e7f0c8edd0510739bd07815ff617578ffa8cc3/lib/ripper-tags.rb#L68-L77).